src/main/java/com/ibm/watson/apis/conversation_with_discovery/rest/ProxyResource.java

```diff
@@ -101,13 +116,14 @@ private MessageRequest buildMessageFromPayload(InputStream body) {
    *         intent confidence is high or the intent is out_of_scope, the response will also contain information from
    *         the discovery service
    */
-  private MessageResponse getWatsonResponse(MessageRequest request, String id) throws Exception {
+  private MessageResponse getWatsonResponse(MessageOptions options) throws Exception {
 
     // Configure the Watson Developer Cloud SDK to make a call to the
     // appropriate conversation service.
 
-    ConversationService service =
-        new ConversationService(API_VERSION != null ? API_VERSION : Constants.CONVERSATION_VERSION);
+    Conversation service =
+        new Conversation(API_VERSION != null ? API_VERSION : Constants.CONVERSATION_VERSION);
```

Comment:
Let's default to `Constants.CONVERSATION_VERSION` no need to use the ternary operator